### ISSUE TITLE
accuracy problem

### ISSUE THREAD

**Issue by beifangbai** [mpapadomanolaki/UNetLSTM#3]
You input two sets of data in four bands to the unet-lstm network, and the result is that F1 is 57.22, but I can only get F1 to 54.1 during the test. Is your result the maximum value obtained from multiple tests? I tested using a 1080ti GPU.
thanks very much.

**Comment by mpapadomanolaki** [comment]
Hello,

Yes i followed a cross validation process, splitting the the training dataset in 5 parts. The training process was performed 5 times, each time using 4 parts as training and 1 part as validation. Then the testing result was the average of the predictions of the 5 models.

**Comment by beifangbai** [comment]
Thank you very much for your answer.

**Comment by beifangbai** [comment]
Hello,
I have another question to ask you. After each training, how do you choose the best model? Is the model with the smallest validation loss selected?
thanks very much.

**Comment by mpapadomanolaki** [comment]
I am plotting the training-vaidation loss graph for the whole training process and I try to choose the point where 2 criteria are met:
1) the validation loss is small 
2) the difference between training and validation loss is also small
